docs/how-to/extract-a-backup.md

```diff
@@ -45,6 +45,22 @@ in the right place before running the command—or see below about the
 `--destination` flag.
 
 
+## Extract the configuration file used to create an archive
+
+<span class="minilink minilink-addedin">New in version 1.7.15</span> Borgmatic
+also stores the configuration file used to create an archive, inside the
+archive itself. This is useful in cases where you've lost your configuration
+file, or you want to see what configuration was used to create a particular
+archive.
+
+To extract the configuration file from an archive, use the `config bootstrap` action:
+
+```bash 
+borgmatic config bootstrap --repository repo.borg --destination /tmp
+```
```

Comment:
I think it's also worth mentioning that:

1. You don't need any existing configuration files on disk in order to use the `bootstrap` feature! (That's why `--repository` is required.)
2. If you don't specify a destination, it will extract configuration files to their original path, silently overwriting any configuration files that may already exist. I think that's important to mention both because it seems like a common way to use the feature and also because it might be a little surprising if someone isn't expecting the overwrite.
3. IMO not all command-line flags of an action need to be documented, but it seems like `--archive` is important to mention..?